cpp/IceDiscovery/greeter/README.md

```diff
@@ -0,0 +1,39 @@
+# IceDiscovery Greeter
+
+This demo illustrates how to configure the IceDiscovery plugin. The IceDiscovery plugin allows a client application
```

Comment:
There is really no configuration anymore, especially when you understand configuration = Ice properties.

how to use the IceDiscovery plugin? how to enable the IceDiscovery plugin?